src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -17,7 +17,8 @@
       "version": "@SDK_PACK_VERSION@",
       "alias-to": {
         "osx-x64": "Microsoft.Android.Sdk.osx-x64",
-        "win-x64": "Microsoft.Android.Sdk.win-x64"
+        "win-x64": "Microsoft.Android.Sdk.win-x64",
+        "linux-x64": "Microsoft.Android.Sdk.linux-x64"
```

Comment:
I don't think we should add this line unless we are also pushing the `Microsoft.Android.Sdk.linux-x64` pack to our NuGet feed.

It doesn't look like we do yet: https://dev.azure.com/azure-public/vside/_packaging?_a=feed&feed=xamarin-impl%40Local